lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/ShowPageProducer.java

```diff
@@ -4944,9 +4937,22 @@ private void createEditTitleDialog(UIContainer tofill, SimplePage page, SimplePa
 				List<String> possOwners = new LinkedList<>();
 				boolean isOwned = page.isOwned();
 				String owner = page.getOwner();
-				possOwners.addAll(simplePageBean.getCurrentSite().getUsers());
 				Set<String> siteUsersCanUpdate = simplePageBean.getCurrentSite().getUsersIsAllowed(SimplePage.PERMISSION_LESSONBUILDER_UPDATE);
-				possOwners.removeAll(siteUsersCanUpdate);
+
+				// Sort the site member list before filling the "possOwners" list
+				List<Member> siteMemberList = new ArrayList<Member>(simplePageBean.getCurrentSite().getMembers());
```

Comment:
Does it make sense to get all members here? Why not a userDirectoryService.getUsers call with only the few users that can update (siteUsersCanUpdate)?

Reply:
Hey Sam, looks like that list is built with all the site members except the ones that have editing permissions.

Reply:
This piece of code has been changed because the list of users should be ordered by name.

Reply:
I see .... Thanks